entrypoint.sh

```diff
@@ -4,14 +4,16 @@ set -e
 
 git remote set-url origin "https://${INPUT_TOKEN}@github.com/${GITHUB_REPOSITORY}"
 
-echo "Getting head sha..."
+echo "Getting HEAD info..."
+
+CURR_SHA=$(git rev-parse HEAD 2>&1) && exit_status=$? || exit_status=$?
```

Comment:
Can we use [github.sha](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context) instead ??

Reply:
Sure thing, I like that. Added.